src/components/Dropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Dispatch, ReactElement, SetStateAction, useState } from 'react'
+import DropDownPicker, { ItemType, ValueType } from 'react-native-dropdown-picker'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled, { useTheme } from 'styled-components/native'
+
+interface DropdownProps<T> {
```

Comment:
Optional: I think its kind of an convention in our code to put the props type directly above the actual component and not above the styled components. I would prefer to keep it that way.

Reply:
didn't know. adjusted